app/src/main/res/layout/fragment_birthday_sign_up.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/white"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    tools:context=".login.sign_up.screens.BirthdaySignUpFragment">
+
+    <TextView
+        android:id="@+id/birthday_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/_24dp"
+        android:layout_marginTop="@dimen/_8dp"
+        android:fontFamily="@font/urbanist_bold"
+        android:layout_alignParentTop="true"
+        android:text="@string/what_is_your_birthday"
+        android:textSize="@dimen/_30sp" />
+
+    <EditText
+        android:id="@+id/et_birthday_day"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:autofillHints="day"
+        android:maxLength="2"
+        android:layout_marginStart="@dimen/_24dp"
+        android:layout_marginTop="@dimen/_24dp"
+        android:hint="@string/_29"
+        android:inputType="numberDecimal"
+        android:textSize="22sp"
```

Comment:
Her edittext'e tek tek property vermek yerine bunları style'a çıkarabilirsin. Ya da daha iyisi, Custom View yazabilirsin.

Reply:
CustomView şu an için biraz zamanımı alır diye düşünüyorum, ilerleyen zamanlarda vaktim kalırsa Custom View ile çıkabilirim

Reply:
Style olarak yapıp tekrardan pr atacağım.